labs-ios-starter/Application/Network/NetworkService.swift

```diff
@@ -159,7 +159,8 @@ class NetworkService {
         }
 
         do {
-            return try decoder.decode(T.self, from: data)
+            let topics = try decoder.decode(T.self, from: data)
+            return topics
```

Comment:
Why this instead of return try? Also, considering this is meant to decode any Decodable type, calling the variable topics isn't exactly semantic